homeassistant/components/uvc/camera.py

```diff
@@ -106,6 +109,22 @@ def motion_detection_enabled(self):
         caminfo = self._nvr.get_camera(self._uuid)
         return caminfo['recordingSettings']['motionRecordEnabled']
 
+    @property
+    def supported_features(self):
+        """Return supported features."""
+        caminfo = self._nvr.get_camera(self._uuid)
```

Comment:
Does this do I/O ? We are not allowed to do any I/O inside properties.

Reply:
Probably. I was just copying what was already done here: https://github.com/home-assistant/home-assistant/blob/59f6bc393687a25b699c81a3d22974b3a605c1cc/homeassistant/components/uvc/camera.py#L103